Misc/NEWS.d/next/Core and Builtins/2024-03-30-19-34-51.gh-issue-117394.qCepkD.rst

```diff
@@ -0,0 +1 @@
+Reduce syscalls for :func:`posixpath.ismount`.
```

Comment:
If I'm a user of `os.path.ismount()`, I might be wondering, "What impact does this have on me? Why should I necessarily care about the number of syscalls?" Something like this would be better, concentrating on the impact that this change has for users of the function:

```suggestion
Speedup :func:`os.path.ismount` on Unix systems by reducing the number of
syscalls.
```